cmd/genji/dbutil/dump.go

```diff
@@ -64,10 +64,86 @@ func Dump(ctx context.Context, db *genji.DB, w io.Writer, tables ...string) erro
 	return err
 }
 
-// dumpTable displays the content of the given table as SQL statements.
-func dumpTable(tx *genji.Tx, tableName string, w io.Writer) error {
+// dumpTableContent displays the content of the given table as SQL statements.
+func dumpTableContent(tx *genji.Tx, w io.Writer, tableName string) error {
+	// Dump schema first.
+	if err := dumpSchema(tx, w, tableName); err != nil {
```

Comment:
The name of the function indicates that it only dumps the table content, not schema. Either we keep the same name as before, or we remove the `dumpSchema` call from this function and call it separately

Reply:
You're right , it's better to keep as before `dumpTable` and no need to make a new function `dumpTableContent`.